Applications/Spire/Include/Spire/Ui/Tag.hpp

```diff
@@ -0,0 +1,43 @@
+#ifndef SPIRE_TAG_HPP
+#define SPIRE_TAG_HPP
+#include <QWidget>
+#include "Spire/Ui/Ui.hpp"
+
+namespace Spire {
+
+  /** Represents a Tag in a TagBox. */
+  class Tag : public QWidget {
+    public:
+
+      /**
+       * Signals that the tag is being clicked to delete.
+       */
+      using DeleteSignal = Signal<void ()>;
+
+      /**
+       * Constructs a Tag.
+       * @param label The text label to display inside of the Tag.
+       * @param parent The parent widget.
+       */
+      explicit Tag(QString label, QWidget* parent = nullptr);
+
+      /** Returns <code>true</code> iff the Tag is deletable by the user. */
+      bool is_read_only() const;
+
+      /**
+       * Sets the read-only state.
+       * @param is_read_only True iff the Tag should be deletable.
```

Comment:
I think this should be: "should not be deletable."

Reply:
Good catch.